integreat_cms/cms/templates/pages/_page_permission_table.html

```diff
@@ -74,11 +74,11 @@
     {% endfor %}
 </div>
 <div class="w-full flex flex-wrap mt-2">
-    {% render_field page_form.publishers %}
+    {% render_field page_form.editors %}
     <button class="btn grant-page-permission mt-2 w-full"
             data-url="{% url 'grant_page_permission_ajax' region_slug=request.region.slug %}"
             data-page-id="{{ page.id }}"
             data-permission="publish">
-        <i icon-name="plus-circle" class="mr-2"></i> {% translate "Add to editors" %}
+        <i icon-name="plus-circle" class="mr-2"></i> {% translate "Add to authors" %}
```

Comment:
```suggestion
        <i icon-name="plus-circle" class="mr-2"></i> {% translate "Add to editors" %}
```
In this particular case, the string was already correct (this is the text shown in the GUI which was fixed earlier)...